Apps/Src/ReadTritium.c

```diff
@@ -9,19 +9,33 @@
 #include "CAN_Queue.h"
 #include "CANbus.h"
 #include "UpdateDisplay.h"
+#include "os_cfg_app.h"
 #include <string.h>
 
 //status limit flag masks
-#define MOTOR_RESTART_THRESHOLD 3	// Number of times to restart before asserting a nonrecoverable error
+#define MASK_MOTOR_TEMP_LIMIT (1<<6) //check if motor temperature is limiting the motor 6
```

Comment:
Just wondering what the 6 in the comment signifies

Reply:
Honestly no clue, I'll remove it.